runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -3283,6 +3298,22 @@ void chpl_comm_pre_task_exit(int all)
     return;
 
   if (all) {
+
+    if (chpl_nodeID == 0) {
+      int i;
+      for (i = 0; i < chpl_numNodes; i++) {
+	if (i != chpl_nodeID) {
```

Comment:
Just noticed that I could just start at 1 instead of 0 and skip this conditional -- thoughts?